### PR TITLE
CRDs helm subchart

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR creates a dedicated **Helm Sub Chart** for Liqo **CRDS**.

This avoids problems with **Helm Chart Size Limitations** encountered with new network API.
https://azure.github.io/azure-service-operator/design/adr-2023-02-helm-chart-size-limitations/

### CODE REVIEW

**Comment by cheina97** [comment]
/ hold

**Comment by cheina97** [comment]
Waiting for https://github.com/liqotech/liqo/pull/1994

File: apis/networking/v1alpha1/connection_types.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	ConnectionConditionStatusFalse ConnectionConditionStatusType = "False"
+	// ConnectionConditionStatusUnknown represents a connection condition that is unknown.
+	ConnectionConditionStatusUnknown ConnectionConditionStatusType = "Unknown"
+)
+
+// ConnectionCondition contains details about state of the connection.
+type ConnectionCondition struct {
+	// Type of the connection condition.
+	// +kubebuilder:validation:Enum="Established";"Pending";"Denied";"Error"
+	Type ConnectionConditionType `json:"type"`
```
Comment (fra98) [inline apis/networking/v1alpha1/connection_types.go:98]:
```suggestion
	// +kubebuilder:validation:Enum="Established";"Pending";"Denied";"Error"
    // +kubebuilder:default="Pending"
	Type ConnectionConditionType `json:"type"`
```

File: deployments/liqo/charts/liqo-crds/values.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
```
Comment (aleoli) [inline deployments/liqo/charts/liqo-crds/values.yaml:82]: These values are pretty useless.

**Review by aleoli — COMMENTED** [review]
You should include the crd chart as a subchart of the main one.

Additionally, we should add the action to publish the chart.

File: deployments/liqo/charts/liqo-crds/values.yaml (hunk 2 above)
Comment (cheina97) [reply deployments/liqo/charts/liqo-crds/values.yaml:82]: I forgot to delete it

**Pull request merged**